core/Cargo.toml

```diff
@@ -54,7 +54,7 @@ slog 							= "2"
 slog-async 						= "2"
 slog-term 						= "2"
 rustc-hash 						= "1.1"
-hocon 							= {version = "0.3", default-features = false}
+hocon 							= {version = "0.3.5", default-features = false}
```

Comment:
Why the limitation to a concrete version? Is there a compatibility issue that we need to keep track of?

Reply:
You're right there is no reason. I was fiddling around a bunch to deserialize the BufferConfig with serde, but I ended up just reading the values one by one instead to avoid cloning/consuming the Hocon object. I'll revert this change!